install/configuring_inventory_file.adoc

```diff
@@ -975,9 +975,21 @@ xref:../install_config/redeploying_certificates.adoc#install-config-redeploying-
 [[advanced-install-configuring-registry-location]]
 == Configuring a Registry Location
 
-If you use an image registry other than the default at
-`registry.redhat.io`, specify the registry in the
-*_/etc/ansible/hosts_* file.
+If you use the default registry at `registry.redhat.io`, you must set the following
+variables:
+
+----
+oreg_url=registry.redhat.io/openshift3/ose-${component}:${version}
```

Comment:
Please make the variable format consistent in this code block. All the parameter values should be enclosed in [angle brackets](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#command-syntax).